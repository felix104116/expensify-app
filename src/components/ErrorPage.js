import React from 'react';
import { Link } from 'react-router-dom';
const ErrorPage = () => (
    <div>
        <p>Error 404</p>
        <Link to="/">Go Home</Link>
    </div>
);
export default ErrorPage;