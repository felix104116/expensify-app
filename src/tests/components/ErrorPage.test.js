import ErrorPage from '../../components/ErrorPage';
import React from 'react';
import {shallow} from 'enzyme';

test('show render error page', () => {
    const wrapper = shallow(<ErrorPage/>);
    expect(wrapper).toMatchSnapshot();
})