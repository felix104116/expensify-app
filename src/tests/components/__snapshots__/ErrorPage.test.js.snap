// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`show render error page 1`] = `
<div>
  <p>
    Error 404
  </p>
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
